joomla/joomla-cms
# Tinymce doesn't need to remove the images basepath

Pull Request for Issue #25015 .

### Summary of Changes
Removed the deletion of the image base path from the upload path.


### Testing Instructions
c/p from https://github.com/joomla/joomla-cms/issues/25015#issuecomment-496568853

Create a sub-directory of "images" under the main images directory.
Create a sub-directory of "testing123" under that directory.

Update the plugin: Editor - TinyMCE
Make sure you are editing tab "Set 0" so Administrator is showing in the groups that use this profile.
Under the "Images Directory" setting, select "images/testing123" (save & close)

> please try all variants

the actual path on the server is "images/images/testing123"
Go under content, and articles and either create a new or edit an existing.

Drag and Drop an image into the content window.

Also change the image and file path in com_media options and try it with different pathes


### Expected result
You should not receive an error and see the image.


### Actual result
You should receive an error the file can not be uploaded, showing the filename as undefined.
